performance/README.md

```diff
@@ -0,0 +1,114 @@
+
+Cluster Autoscaling setup is dependent on the cloud provider and/or the Kubernetes distribution. For example:
+- AWS: The [Cluster Autoscaler](https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html) can be installed as a DaemonSet.
+- GCP: The [Cluster Autoscaler](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler) can be enabled on the cluster.
+- OpenShift: The [Cluster Autoscaler](https://docs.openshift.com/container-platform/latest/machine_management/applying-autoscaling.html) can be managed through the OpenShift web console or CLI.
+- Azure AKS: The [Cluster Autoscaler](https://learn.microsoft.com/en-us/azure/aks/cluster-autoscaler) can be enabled directly from the Azure portal or via Azure CLI.
+
+Cluster Autoscaling is conceptually simple; when trainee pods are created, that do not fit in the existing cluster, the autoscaler component will send messages to the infrastructure provider to create new nodes to accommodate them.  When the trainee pods are deleted, the autoscaler will remove the nodes.
+
+For the best performance, Howso Platform workloads should be split into two node pools.  One for the core services, NATS and datastores, and one for the trainee pods.  This sets up the core services to run permently without interference from the trainee pods, and allows the trainee pods to be scaled independently of the core services.  The core node pool should have a permenant minimum number of nodes, optionally autoscaling (an extra node or 2) to allow for certain services to [scale](#core-service-horizontal-pod-autoscaler) under heavy load.  The trainee node pool should be autoscaling, potentially to a minimum of 0 nodes and up to the largest capacity required for the heaviest workloads.  Only trainee pods should run on the trainee node pool. 
+
+#### Node Pool Configuration
+
+When configuring autoscaling configuration, it may be useful to refer to the [multi-node setup](../prereqs/README.md#multi-node-k3d-cluster) which demonstrates the use of labels and taints (in a local setup) to control pod scheduling across multiple nodes.
+
+##### Core Services Nodes
+
+The core services of Howso Platform and its dependent charts should run on 2 or 3 dedicated cluster nodes utilizing a [node label](../prereqs/README.md#multi-node-k3d-cluster), to keep trainee pods from starting on them.
+
+```sh
+kubectl label nodes k3d-platformk8s-server-0 howso.com/allowWorkers=False --overwrite
+```
+
+Note, since some core services are set up to [auto-scale](#core-service-horizontal-pod-autoscaler) the core service nodes can be set to to autoscale to accomodate any increased demand under load.  They should not be allowed to scale below the minimum number of nodes required to run the core services.
```

Comment:
Are the corresponding HPA settings in the Helm chart we distribute?  What are they?  What are the acceptable minimums?

Reply:
Yep - they are mentioned in the linked section